source/scss/_pf-basic-tags.scss

```diff
@@ -24,6 +24,13 @@ limitations under the License.
   h4 &,
   h5 &,
   h6 & {
+    // Size of the tag is relative to the font size of the title
     font-size: $font-size-relative-xs;
   }
+
+  // This anchor targets the close icon in dismissable tags
+  a{
+    color: $pf-color-white;
+    margin-left: $pf-spacer-xxxs;
```

Comment:
Not sure if this is a good solution to the alignment issue, but setting `vertical-align: text-bottom;` here (or on the child `<span>`) makes the X vertically centered in the tag block, but also increases the height of the tag block.